### DISCUSSION TITLE
More privacy friendly alternative to Google Analytics

### DISCUSSION THREAD

**Discussion by T0nio** [forem/forem#17200] [Icebox]
Ability to configure a more privacy friendly alternative to Google Analytics. For instance Plausible (it's self hostable, so it would be very nice to be able to choose our server). 

Another alternative would be to add the ability to add custom HTML code in the header of pages to add our analytics snipet (and would be usable for other custom code)

**Comment by ellativity** [comment]
+1 for [Plausible](https://plausible.io/) etc.

**Comment by JennieOcken** [comment]
@T0nio 

Thank you so much for your suggestion.  We have looked into a couple different alternatives to GA that we think would be more friendly.  We have two different paths towards this.

1) Explicitly support other Analytics providers ([internal reference](https://github.com/forem/forem-internal-eng/issues/139)).   My plan into evaluate this approach later in 2022 / early 2023 after we get through some of our more fundamental projects.

2) Allow a Creator to add their own JavaScript or HTML to the header of the page.  We actually have a [task open](https://github.com/forem/forem/issues/16333) for this right now.  It is on our "high impact, small effort" list.  Which means we slip in these tasks as our team has time or look for contributors to help us out with these features.

Given that, I am going to put this request in our "icebox".  Which simply means "I love this idea but we aren't going to tackle it right now and I will open this up sometime in the future.

Thanks!
